Dfe.Academies.External.Web/Pages/WhatAreYouApplyingToDo.cshtml.cs

```diff
@@ -56,7 +57,7 @@ public async Task<IActionResult> OnPostAsync()
             var _draftConversionApplication = new ConversionApplication
             {
                 ApplicationType = applicationTypeSelected,
-                UserEmail = "Auth user"
+                UserEmail = User.FindFirst(ClaimTypes.Email)?.Value
```

Comment:
I'm not actually sure that this field should in in the model, I don't have this in the back end. I think we just want the Contributor.

Reply:
Can decide later when doing the integration work though.

Reply:
rip it out I say

Reply:
Removed